package.json

```diff
@@ -10,7 +10,7 @@
     "url": "github.com/SamVerschueren"
   },
   "scripts": {
-    "test": "xo && ava"
+    "test": "npx xo && npx ava"
```

Comment:
Undo

Reply:
Sorry about that.  Undone here: 1d43436f12f7be